webpack.config.js

```diff
@@ -0,0 +1,40 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: path.resolve(__dirname, 'src/index.js'),
+  devServer: {
+    static: {
+      directory: path.resolve(__dirname, 'dist'),
+    },
+    port: 3000,
+    open: true,
+    hot: true,
+    compress: true,
+    historyApiFallback: true,
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/template.html',
+    }),
+  ],
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+  },
+  optimization: {
```

Comment:
please consider giving a specific name for your output file. so webpack default not be generated, you have two js files on you dist/ folder (runtime, main) seeing you have no code splitting. you should have one bundle file only please add the output path and build it to have a single js bundle in the dist/ folder

Reply:
👍🏻